.github/workflows/ci.yml

```diff
@@ -35,6 +35,9 @@ jobs:
           key: ${{runner.os}}-go-${{hashFiles('**/go.sum')}}-test
           restore-keys: |
             ${{runner.os}}-go-
+      - name: Check Cheatsheet
+        run: |
+          go run scripts/cheatsheet/main.go check
```

Comment:
Very similar to `lazygit`'s . Still left it inside `CI` job as it seems fit the context. Let me know if you think i should separate the jobs.